NOTICE

```diff
@@ -0,0 +1,24 @@
+Apache Baremaps (incubating)
+Copyright 2022-2023 The Apache Software Foundation.
+
+Baremaps development team
+Copyright 2020-2022.
+
+This product includes protocol buffers definitions copied from OSMPBF.
+Copyright (c) 2010 Scott A. Crosby. <scott@sacrosby.com>
+MIT License.
+Please visit the following URLs for the licensing information:
+https://github.com/openstreetmap/OSM-binary/issues/18
+https://github.com/openstreetmap/OSM-binary/pull/35
+
+This product includes software derived from Palantir Streams.
+Copyright 2015 Palantir Technologies, Inc.
+Apache License, Version 2.0
+Please visit the following URL for the full text of the Palantir license:
+https://github.com/palantir/streams/blob/develop/LICENSE
+
+This product includes software derived from Planetiler.
+Copyright 2021 Michael Barry and Planetiler Contributors.
+Apache License, Version 2.0
+Please visit the following URL for the full text of the Planetiler license:
+https://github.com/onthegomap/planetiler/blob/main/LICENSE
```

Comment:
You seem to have forgotten to declare it in the LICENSE file.

Reply:
Should I copy the NOTICE in the LICENSE file? Isn't it redundant?

Reply:
In the `LICENSE` file, we usually just have a short summary

https://github.com/apache/skywalking/blob/60b061f5f9f48a7dd0aa238f9218503c74882ad0/LICENSE#L215_L227

Here are docs https://infra.apache.org/licensing-howto.html#bundled-vs-non-bundled

Reply:
If you are source code dependent and the dependent code used license is the AL2 license, the `NOTICE` file needs to contain the `NOTICE` of the other party. 

Except for the AL2 license, I have not seen other licenses that require modifying the `NOTICE` file.